File: Tutorial4-4ComponentBuildersMultipleModules/src/main/java/com/example/tutorial4_4componentbuildersmultiplemodules/ToastMaker.java
package com.example.tutorial4_4componentbuildersmultiplemodules;

import android.app.Application;
import android.widget.Toast;

public class ToastMaker {

    private Application application;

    public ToastMaker(Application application) {
        this.application = application;
    }

    public void showToast(String message) {
        Toast.makeText(application, message, Toast.LENGTH_SHORT).show();
    }


}

File: Tutorial4-4ComponentBuildersMultipleModules/src/main/java/com/example/tutorial4_4componentbuildersmultiplemodules/model/ToastMaker.kt
package com.example.tutorial4_4componentbuildersmultiplemodules.model

import android.app.Application
import android.widget.Toast

class ToastMaker(private val application: Application) {

    fun showToast(message: String) {
        Toast.makeText(application, message, Toast.LENGTH_SHORT).show()
    }

}